### DISCUSSION TITLE
Telemetry mirror + HC-05 bt module on PA9 pin doesn't send data to "Telemetry Viewer" on android

### DISCUSSION THREAD

**Discussion by Al05ex** [OpenI6X/opentx#398] [Q&A]
Hy, i have configured and wired the HC-05 same as in Rakish videos https://youtu.be/13xWNAUrR-k?si=lZAslgi9HZ5JiAQu , enabled telemetry mirror and set baudrate on radio to 400k, then 1.8M then 115200.. but none of them actually sent me any data to the phone. The bluetooth connection is working, i can connect to the device, but i don't get data in "Telemetry viewer" on android, nor in Mission Planner inav. ELRS Telemetry  is working betwen the tx and the rx, i get all sensors, and i can view them on my flysky fsi6x screen. Do i need a special LUA script to enable telemetry mirror to do something ? I have measured at some point that pa9 pin, and i got 3.3v on the multimeter, that is wired to hc05 rx pin.
I have looked in all the settings, but i can't get it to show me any data from telemtry mirror pin.. Trainer port is working when connected to the pc with usb, At commands worked for the hc module, when i configured it through arduino ide, and set buadrate to 115200..
I don't know what else to do..

**Comment by ajjjjjjjj** [comment]
AUX Serial was broken on 1.11.x release. It is now fixed on master, and will be released in 1.12.

**Comment by Al05ex** [comment]
Ok, thanks for your answer! I have solved this by using a previous version,
back then. But i am glad that you have fixed the issue!

Although, i don t know exactly what i did wrong, but when i have updated
the firmware on the STM microcontroller, the version changed if i watched
the settings tab, but the upload year was still the previous one. And the
model settings didn t erase. But it worked after that.

joi, 12 sept. 2024, 15:21 Janek ***@***.***> a scris:

**Comment by ajjjjjjjj** [comment]
Recent versions do not change user configuration so it stays as expected.

**Comment by ajjjjjjjj** [comment]
Resolved on 1.12.0 release
